policyengine_us/parameters/gov/states/vt/tax/income/credits/renter/rate/non_taxable_ss.yaml

```diff
@@ -0,0 +1,15 @@
+description: Vermont uses this rate of non-taxable social security benefits for renter credit.
+values:
+  2021-01-01: 0.75
+metadata:
+  unit: /1
+  period: year
+  label: Vermont renter credit non-taxable social security rate
+  reference:
+    # This rate only appears in the tax form and tax form instruction.
```

Comment:
need to file an issue in policyengine-us to consolidate the legal code and the tax form